test/integration/integration_suite_test.go

```diff
@@ -157,3 +171,31 @@ func canCreateOpenshiftNamespaces() bool {
 
 	return canCreateOpenshiftNamespacesResult
 }
+
+func applyManagedClusterSetBinding() error {
+	yaml := `
+apiVersion: cluster.open-cluster-management.io/v1beta2
```

Comment:
Why not use the dynamic client for this?

Reply:
Dale wants to have a namespace we can control(able to delete). but open-cluster-management-global-set we cannot delete. I think to me both ok. policy-test we use as `Useranmespace`. This is used as constant

Reply:
@mprahl Isn't making a comment here about the namespace. He's making a similar point as I did about using the Golang Kubernetes client rather than creating a temporary file.

Reply:
@dhaiducek  you're right!!